src/components/MyKiva/BadgeContainer.vue

```diff
@@ -154,10 +154,6 @@ onMounted(() => {
 	@apply tw-h-full tw-w-full;
 }
 
-.invisible-badge :deep(img) {
-	@apply tw-invisible;
```

Comment:
Why is this no longer needed?

Reply:
So there is no longer an image that needs to be hidden? Was the styling not really doing anything?